src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,44 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ARRAY_SIZE = 10;
+    private K[] keys;
+    private V[] values;
+    private int sizeOfStorage = 0;
```

Comment:
```suggestion
    private int sizeOfStorage;
```

class parameter of type `int` initialized to `0` by default